mbed_lstools/darwin.py

```diff
@@ -152,11 +165,7 @@ def _volumes(self):
 
         for usb_controller in usb_controllers:
             ioreg_usb = subprocess.Popen(['ioreg', '-a', '-r', cmp_par, usb_controller, '-l'], stdout=subprocess.PIPE)
-            try:
-                usb_tree = plistlib.readPlist(ioreg_usb.stdout)
-            except:
-                usb_tree = []
-            ioreg_usb.wait()
+            usb_tree = _plist_from_popen(iored_usb)
```

Comment:
Ah, CI beat me to it :P Lil typo here `iored_usb -> ioreg_usb`

Reply:
I had that change, forgot to push it.